index.html

```diff
@@ -39,43 +41,47 @@ <h3 class="font-bold mb-2 text-xl">
             Firefox-lisäosista.
           </p>
           <p class="mb-2">
-            <a class="underline hover:bg-yellow-100" href="#">Jaksomuikkarit</a>
+            <a class="italic underline hover:text-red-400" href="#">Jaksomuikkarit</a>
           </p>
           <p><em>Player</em></p>
         </article>
 
         <h2 class="font-bold mb-4 text-xl">Menneet jaksot</h2>
 
-        <article class="-m-2 p-2 rounded hover:bg-yellow-100">
-          <a href="#">
-            <h3 class="font-bold mb-2 text-lg underline">#2 &ndash; ?</h3>
-            <footer>
-              <p class="mb-2">
-                <span class="sr-only">Julkaistu</span>
-                <time datetime="2021-03-27">27.3.2021</time>
-              </p>
-            </footer>
-            <p>Jaksokuvaus...</p>
-          </a>
-        </article>
-
-        <hr aria-hidden="true" class="my-6" />
+        <div class="flex flex-wrap">
+          <article class="w-2/4">
+            <div class="mr-3 p-3 rounded bg-white shadow-md hover:shadow-lg transition">
+              <a href="#">
```

Comment:
When hovering the link, the mouse cursor looks like a pointer, like it should. But if you move the cursor between the h3 and the footer (publication date), the mouse cursor reverts back to the default. Looks buggy.

One way to fix is to make the link have `display: block`. You could also move the `p-3` class to this element. (Why? Try moving the cursor to the edge of a card. The card gets a heavier shadow even though you are not yet hovering the link, which looks buggy. Does this make sense?)